### PR TITLE
fix: remove subdirectories, not root path for migration command

### PR DESCRIPTION
**Pull request by istae**
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/2746)

### CODE REVIEW

File: cmd/bee/cmd/db.go [hunk 1]
```diff
@@ -230,3 +230,24 @@ func dbNukeCmd(cmd *cobra.Command) {
 	c.Flags().Duration(optionNameSleepAfter, time.Duration(0), "time to sleep after the operation finished")
 	cmd.AddCommand(c)
 }
+
+func removeContent(path string) error {
+	dir, err := os.Open(path)
```
Comment (acud) [inline cmd/bee/cmd/db.go:235]: this is a bit misleading. i believe that if you use this you would then get everything that sits in the directory in which `path` is contained. did you try this manually? i would consider using [filepath.Walk](https://pkg.go.dev/path/filepath#Walk)

Comment (istae) [reply cmd/bee/cmd/db.go:235]: yeah, tried manually, root folder is preserved

**Review by acud — APPROVED** [review]
thanks!

**Review by notanatol — APPROVED** [review]

**Review by mrekucci — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/2746)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @acud, @istae, and @mrekucci)

___
*[cmd/bee/cmd/db.go, line 248 at r2](https://reviewable.io/reviews/ethersphere/bee/2746#-Ms_URqWETfpaAbeGRin:-Ms_URqWETfpaAbeGRio:b-152phs) ([raw file](https://github.com/ethersphere/bee/blob/b76e53ff61f21b6a6b3d838b3cf068d1ccb260c0/cmd/bee/cmd/db.go#L248)):*
> ```Go
> 	for _, sub := range subpaths {
> 		err = os.RemoveAll(filepath.Join(path, sub))
> 		if err != nil {
> ```

It is desirable to return on the first error? Wouldn't be better to collect all the errors (and continue the execution) and return flattened multierror?

**Review by acud — APPROVED** [review]

**Pull request merged**